modules/drivers/snowflake/src/metabase/driver/snowflake.clj

```diff
@@ -517,10 +517,10 @@
   (sql-jdbc.execute/do-with-connection-with-options
    driver database nil
    (fn [^java.sql.Connection conn]
-     (with-open [stmt (.prepareStatement conn "show parameters like 'TIMEZONE';")
+     (with-open [stmt (.prepareStatement conn "show parameters like 'TIMEZONE' in user;")
                  rset (.executeQuery stmt)]
        (when (.next rset)
-         (.getString rset "default"))))))
```

Comment:
`value` gives us actual timezone that user is using (that can change in between syncs). `default` is the default for every deployment and does not have to correspond to what's in actual use.